Modules/Office365DSC/DSCResources/MSFT_TeamsMemberSettings/MSFT_TeamsMemberSettings.psm1

```diff
@@ -0,0 +1,230 @@
+
+        [Parameter()]
+        [ValidateSet("Present", "Absent")]
+        [System.String]
+        $Ensure = "Present",
+
+        [Parameter(Mandatory = $true)]
+        [System.Management.Automation.PSCredential]
+        $GlobalAdminAccount
+    )
+
+    if ('Absent' -eq $Ensure)
+    {
+        throw "This resource cannot delete Managed Properties. Please make sure you set its Ensure value to Present."
+    }
+
+    Test-TeamsServiceConnection -GlobalAdminAccount $GlobalAdminAccount
+
+    $team = Get-Team |  Where-Object {$_.DisplayName -eq $TeamName}
+    if ($null -eq $team)
+    {
+        throw "Team with Name $TeamName doesnt exist in tenant"
+    }
+
+    $CurrentParameters = $PSBoundParameters
+    $CurrentParameters.Remove("TeamName")
+    $CurrentParameters.Add("GroupId", $team.GroupId)
+    $CurrentParameters.Remove("GlobalAdminAccount")
+    $CurrentParameters.Remove("Ensure")
+    Set-TeamMemberSettings @CurrentParameters
+}
+
+function Test-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Boolean])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $TeamName,
+
+        [Parameter()]
+        [System.Boolean]
+        $AllowCreateUpdateChannels,
+
+        [Parameter()]
+        [System.Boolean]
+        $AllowDeleteChannels,
+
+        [Parameter()]
+        [System.Boolean]
+        $AllowAddRemoveApps,
+
+        [Parameter()]
+        [System.Boolean]
+        $AllowCreateUpdateRemoveTabs,
+
+        [Parameter()]
+        [System.Boolean]
+        $AllowCreateUpdateRemoveConnectors,
+
+        [Parameter()]
+        [ValidateSet("Present", "Absent")]
+        [System.String]
+        $Ensure = "Present",
+
+        [Parameter(Mandatory = $true)]
+        [System.Management.Automation.PSCredential]
+        $GlobalAdminAccount
+    )
+
+    Write-Verbose -Message "Testing Team member settings for $TeamName"
+    $CurrentValues = Get-TargetResource @PSBoundParameters
+
+
+
+    return Test-Office365DSCParameterState -CurrentValues $CurrentValues `
+        -DesiredValues $PSBoundParameters `
+        -ValuesToCheck @("AllowCreateUpdateChannels", `
+            "AllowDeleteChannels", `
+            "AllowAddRemoveApps", `
+            "AllowCreateUpdateRemoveTabs", `
+            "AllowCreateUpdateRemoveConnectors", `
+            "Ensure"
+    )
+}
+
+function Export-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.String])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $TeamName,
+
+        [Parameter(Mandatory = $true)]
+        [System.Management.Automation.PSCredential]
+        $GlobalAdminAccount
+    )
+    Test-TeamsServiceConnection -GlobalAdminAccount $GlobalAdminAccount
+    $result = Get-TargetResource @PSBoundParameters
+    $result.GlobalAdminAccount = Resolve-Credentials -UserName $GlobalAdminAccount.UserName
+    $content = "        TeamsMemberSettings " + (New-GUID).ToString() + "`r`n"
```

Comment:
Ensure you add logic block in the Export-O365Configuration method

Reply:
Please double check what I added in this script